packs/data/feat-effects.db/stance-inspiring-marshal-stance-critical.json

```diff
@@ -0,0 +1,56 @@
+{
+    "_id": "ymnWPTa0wQuDaQkS",
+    "img": "systems/pf2e/icons/features/feats/inspiring-marshal-stance.webp",
+    "name": "Stance: Inspiring Marshal Stance (Critical)",
```

Comment:
These should be condensed into one effect with a ChoiceSet for success or critical success, and the radius should be set by that choice.

Reply:
I'll give it a shot - if there's a good example I could refer against that would be appreciated.

Reply:
Maybe check out "Spell Effect: Clinging Ice" for the ChoiceSet.

Reply:
That was much cleaner - updated